jalali_date/widgets.py

```diff
@@ -21,7 +21,7 @@ def media(self):
         return forms.Media(js=[static(path) for path in js], css=css)
 
     def __init__(self, attrs=None, format=None):
-        final_attrs = {'class': 'jalali_date-date', 'size': '10'}
+        final_attrs = {'data-jdp': {}, 'data-jdp-only-date': {}}
```

Comment:
Please keep the `class` attribute.

Reply:
dad7add67f7ca7d0c470e0ffc24c8e2ec56b30d8